### ISSUE TITLE
Uncaught RangeError: offset is outside the bounds of the DataView

### ISSUE THREAD

**Issue by oksanassss** [RainwayApp/rainway-sdk-web-demo#25]
I runned app on chrome and another  firefox window.  When ı tried Connect to peer I'm geting DataView error.  

```Uncaught RangeError: offset is outside the bounds of the DataView```
![image](https://user-images.githubusercontent.com/89344313/183359097-0222d278-91b2-46b5-92e8-1fe3d1d878b9.png)

**Comment by lynn** [comment]
Hi @oksanassss!

The Web Demo is really meant for connecting to a host app like [host-example-gui](https://github.com/RainwayApp/rainway-sdk-csharp-examples/tree/main/host-example-gui) or your own application. This could admittedly be made a little clearer. That said, this RangeError definitely looks like a bug of some sort.

Are you using https://webdemo.rainway.com/ with `Built on July 27, 2022 at 11:15 PM / Web SDK 0.4.0` in the top right corner, in both browsers?

When connecting browser-to-browser, how are you obtaining a Peer ID to connect to? I notice the Web Demo doesn't show its own Peer ID after connecting to Rainway (this, too, is a shortcoming we might have to address).

**Comment by oksanassss** [comment]
yes I'm using  ``` Web SDK 0.4.0 ``` .  ı Know We cant see peerId on browser.  I added code in  ```demo.tsx``` for see it. 

```
   let conn = await rainway.connect({
          apiKey,
          externalId: "web-demo-react",
        });
        const peerId = conn.id;
        console.log("peerId", peerId); // ı can see peerId
```

**Comment by oksanassss** [comment]
can ı open different webpage for connection betwen their or must ı run [host-example-gui](https://github.com/RainwayApp/rainway-sdk-csharp-examples/tree/main/host-example-gui) ?
